Translations update from Weblate

ubports-weblate opened the pull request:
Translations update from [Weblate](https://translate.ubports.com/projects/ubports/morph-browser/) for UBports/Morph Browser.

UniversalSuperBox reviewed (changes requested):
@Flohack74, this translation seems invalid. https://github.com/ubports/morph-browser/pull/482/files#diff-0a4a9ca503c41c5f2a54de36293d8863d6ef72d42476226fa0b9eddf52bc94e9R499

dobey commented:
> @Flohack74, this translation seems invalid. https://github.com/ubports/morph-browser/pull/482/files#diff-0a4a9ca503c41c5f2a54de36293d8863d6ef72d42476226fa0b9eddf52bc94e9R499

Can you just "fix" it on weblate and let it update this branch so that the broken translation isn't there?

weblate committed:
Translated by Florian Leeber <flori@bin.org.in> using Weblate (Chinese (Simplified)) - 97.9% (240 of 245 strings)

Translated by Enceka <enceka@yeah.net> using Weblate (Chinese (Simplified)) - 98.3% (241 of 245 strings)

Translated by Enceka <enceka@yeah.net> using Weblate (Chinese (Simplified)) - 98.3% (241 of 245 strings)

Translated by Enceka <enceka@yeah.net> using Weblate (Chinese (Simplified)) - 98.3% (241 of 245 strings)

Translated by Enceka <enceka@yeah.net> using Weblate (Chinese (Simplified)) - 98.3% (241 of 245 strings)

Translated by Enceka <enceka@yeah.net> using Weblate (Chinese (Simplified)) - 98.3% (241 of 245 strings)

Translated by Milan Korecký <milan.korecky@gmail.com> using Weblate (Czech) - 100.0% (245 of 245 strings)

Translated by M.Sugahara <equaaqua@hotmail.com> using Weblate (Japanese) - 63.6% (156 of 245 strings)

Translated by louies0623 <louies0623@gmail.com> using Weblate (Chinese (Traditional)) - 98.7% (242 of 245 strings)

Translated by Milan Korecký <milan.korecky@gmail.com> using Weblate (Czech) - 100.0% (245 of 245 strings)

Translated by Daniel Frost <one@frostinfo.de> using Weblate (German) - 100.0% (245 of 245 strings)

Translated by Enceka <enceka@yeah.net> using Weblate (Chinese (Simplified)) - 91.8% (225 of 245 strings)

Translated by dano6 <dano.kutka@gmail.com> using Weblate (Slovak) - 100.0% (245 of 245 strings)

Flohack74 commented:
Should be fixed now...
